packages/hash/api/src/db/postgres/client.ts

```diff
@@ -81,6 +82,83 @@ export class PostgresClient implements DbClient {
     return requireTransaction(this.conn);
   }
 
+  createDataType(_params: {
```

Comment:
I'm guessing the plan is to fully delete the PostgresClient when we've migrated so this is just a temporary state?

Reply:
If so, even though this is exceptionally temporary, I think it'd still be nice to try and leave the codebase in a slightly better intermediate state. 

Could perhaps be fixed by either changing the Error messages, or just adding a TODO explaining what the intention is. (Would be helpful if someone else picks up the next piece of work for example)

Reply:
I will change the exception text to reflect this :)